.github/workflows/deploy.yml

```diff
@@ -154,6 +156,7 @@ jobs:
       - uses: actions/upload-artifact@v3
         with:
           path: ./wheelhouse/*.whl
+          name: aer-deploy-gpu-build-cuda11-${{ matrix.os }}
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Deployed here.  Also, it's still `upload-artifact@v3.`

Reply:
Bumped to v4 and added "separate" in d9d7fb78